index.html

```diff
@@ -0,0 +1,748 @@
+          many, including Dr. Guth, wondered whether it could ever be proved.
+        </p>
+        <p>
+          If corroborated, Dr. Kovac’s work will stand as a landmark in science
+          comparable to the recent discovery of dark energy pushing the universe
+          apart, or of the Big Bang itself. It would open vast realms of time
+          and space and energy to science and speculation.
+        </p>
+        <p>
+          Confirming inflation would mean that the universe we see, extending 14
+          billion light-years in space with its hundreds of billions of
+          galaxies, is only an infinitesimal patch in a larger cosmos whose
+          extent, architecture and fate are unknowable. Moreover, beyond our own
+          universe there might be an endless number of other universes bubbling
+          into frothy eternity, like a pot of pasta water boiling over.
+        </p>
+      </section>
+      <section class="article-section">
+        <h3>‘As Big as It Gets’</h3>
+        <p>
+          In our own universe, it would serve as a window into the forces
+          operating at energies forever beyond the reach of particle
+          accelerators on Earth and yield new insights into gravity itself. Dr.
+          Kovac’s ripples would be the first direct observation of gravitational
+          waves, which, according to Einstein’s theory of general relativity,
+          should ruffle space-time.
+        </p>
+        <p>
+          Marc Kamionkowski of Johns Hopkins University, an early-universe
+          expert who was not part of the team, said, “This is huge, as big as it
+          gets.”
+        </p>
+        <p>
+          He continued, “This is a signal from the very earliest universe,
+          sending a telegram encoded in gravitational waves.”
+        </p>
+        <p>
+          The ripples manifested themselves as faint spiral patterns in a bath
+          of microwave radiation that permeates space and preserves a picture of
+          the universe when it was 380,000 years old and as hot as the surface
+          of the sun.
+        </p>
+        <p>
+          Dr. Kovac and his collaborators, working in an experiment known as
+          Bicep, for Background Imaging of Cosmic Extragalactic Polarization,
+          reported their results in a scientific briefing at the Center for
+          Astrophysics here on Monday and in a set of papers submitted to The
+          Astrophysical Journal.
+        </p>
+      </section>
+
+      <section id="infaltion-section">
+        <h3 class="align-left">The Theory of Inflation</h3>
+        <p class="align-left caption">
+          Astronomers have found evidence to support the theory of inflation,
+          which explains how the universe expanded so uniformly and so quickly
+          in the instant after the Big Bang 13.8 billion years ago.
+        </p>
+        <img src="images/inflation-720.png" alt="inflation image" />
```

Comment:
- [ ] You shouldn't use screenshots of the original page as shortcut to bypass requirements. Please consider building the 'Theory of Inflation' section by using grid, flex or float.

Reply:
Thank you for your helpful review. We worked on this and updated our project according to the required changes.